c2corg_ui/static/menu-discourse/menu.html

```diff
@@ -0,0 +1,190 @@
+    </div>
+
+    <ul class="footer-nav">
+      <li>
+        <a href="#" url="articles/106727/en/contact-us" class="menu-link">
+          <span class="lang en">Contact</span>
+          <span class="lang fr">Contact</span>
+          <span class="lang it menu-text">Contatto</span>
+          <span class="lang de menu-text">Kontakt</span>
+          <span class="lang eu menu-text">Kontaktua</span>
+          <span class="lang ca menu-text">Contacte</span>
+          <span class="lang es menu-text">Contacto</span>
+        </a>
+      </li>
+      <span class="bullet">&bull; </span>
+      <li>
+        <a href="#" url="articles/121965" class="menu-link">
+          <span class="lang en">Credits</span>
+          <span class="lang fr">Crédits</span>
+          <span class="lang it menu-text">Credits</span>
+          <span class="lang de menu-text">Kredite</span>
+          <span class="lang eu menu-text">Credits</span>
+          <span class="lang ca menu-text">Crèdits</span>
+          <span class="lang es menu-text">Créditos</span>
+        </a></li>
+      <span class="bullet">&bull; </span>
+      <li><a href="#" url="articles/106731" class="menu-link">
+          <span class="lang en">Terms of use</span>
+          <span class="lang fr">Conditions d'utilisation</span>
+          <span class="lang it menu-text">Condizioni d'uso</span>
+          <span class="lang de menu-text">Nutzungsbedingungen</span>
+          <span class="lang eu menu-text">Erabilera-baldintzak</span>
+          <span class="lang ca menu-text">Condicions d'ús</span>
+          <span class="lang es menu-text">Condiciones de uso</span>
+        </a></li>
+      <span class="bullet">&bull; </span>
+      <li>
+        <a href="#" url="articles/106728" class="menu-link">
+          <span class="lang en">Content license</span>
+          <span class="lang fr">Licences de contenu</span>
+          <span class="lang it menu-text">Licenze dei contenuti</span>
+          <span class="lang de menu-text">Inhalt Lizenz</span>
+          <span class="lang eu menu-text">Edukiaren lizentziak</span>
+          <span class="lang ca menu-text">Llicències del contingut</span>
+          <span class="lang es menu-text">Licencias de los contenidos</span>
+        </a>
+      </li>
+    </ul>
+
+    <div class="social-networks">
+      <div class="icons">
+        <a href="https://twitter.com/camptocamporg"><img src="#" url="static/img/social/twitter.png" class="src"></a>
+        <a href="https://plus.google.com/104270548458536561874/posts"><img src="#" url="static/img/social/google-plus.png" class="src"></a>
+        <a href="https://www.facebook.com/camptocamp.org/"><img src="#" url="static/img/social/facebook.png" class="src"></a>
+      </div>
+    </div>
+
+  </footer>
+
+  <script>
+
+    var locale = I18n.locale;
+    var menu = document.getElementById('sidemenu');
+    var translations = menu.querySelectorAll('.lang');
+    var baseUrl = '//www.demov6.camptocamp.org/';
```

Comment:
Do you think `` var baseUrl = '//www.demov6.camptocamp.org/';`` could be placed at the beginning of the file? This way it would be more obvious what the ``baseUrl`` is and make admin users easily figure out if it is OK.